app.js

```diff
@@ -47,6 +47,7 @@ const cartOverlay = document.querySelector(".cart-overlay");
 
   const closeCartIcon = document.querySelector(".close-cart");
   closeCartIcon.addEventListener("click", hideCartHandler);
+  cartOverlay.addEventListener("click", hideCartHandler);
```

Comment:
This means that if I click any where on cart it will also get closed but everything is included in cartOverlay. Isn't it?

Reply:
Yes it is so.

Reply:
There is a problem too with this functionality , if i  click any where on the screen either cart or cartOverlay , cart closes immediately , but what if user wants to clear the cart ? what if user wants to remove an item ? cart closes in these situations too  , how can i fix it?